package.json

```
@@ -150,6 +150,13 @@
           ".yaml"
         ]
       }
+    ],
+    "commands": [
+      {
+        "command": "jbang.apache.camel.routes",
+        "title": "Camel: Create a Camel Route using Yaml DSL",
```

```suggestion
        "title": "Create a Camel Route using Yaml DSL",
```

and then use the category `Camel`